lib/ex_webrtc/dtls_transport.ex

```diff
@@ -433,5 +438,15 @@ defmodule ExWebRTC.DTLSTransport do
     %{state | remote_cert: cert, remote_base64_cert: base64_cert, remote_fingerprint: fingerprint}
   end
 
+  defp flush_buffered_remote_rtp_packets(%{buffered_remote_rtp_packets: []} = state), do: state
```

Comment:
This clause is unnecessary. With an empty list, the `for` will work as expected.